server/models/User.js

```diff
@@ -48,10 +59,40 @@ export default class User {
     return Promise.resolve(this.strip());
   }
 
+  async login() {
+    try {
+      const authQuery = `SELECT * FROM authentication WHERE username = '${this.username}'`;
```

Comment:
So instead of performing 2 SQL queries,
You could select from both tables and join

Something like:
```sql
SELECT users.id, authentication.password, authentication.username, users.fullName, users.email FROM authentication INNER JOIN users 
ON users.authId = authentication.authId 
WHERE authentication.username = '${this.username}'
```

I didn't test this though so may need modification

Reply:
Ok, I will.